source/docs/controls-overviews/control-system-hardware.rst

```diff
@@ -1,7 +1,7 @@
 .. include:: <isonum.txt>
 
-FRC Control System Hardware Overview
-====================================
+Control System Hardware Overview
```

Comment:
Should this be Hardware Component Overview?

Reply:
I agree that would match the software, but this has been the title for a bit and I didn't feel right changing it without other opinions.  @Kevin-OConnor thoughts?

Reply:
Kevin said it was cool in slack @jasondaming